tools/package-tools/build-release.ts

```diff
@@ -26,14 +29,73 @@ export function composeRelease(packageName: string) {
   inlinePackageMetadataFiles(packagePath);
 
   copyFiles(packagePath, '**/*.+(d.ts|metadata.json)', join(releasePath, 'typings'));
-  copyFiles(bundlesDir, `${packageName}.umd?(.min).js?(.map)`, join(releasePath, 'bundles'));
+  copyFiles(bundlesDir, `*.umd?(.min).js?(.map)`, join(releasePath, 'bundles'));
```

Comment:
How about something like:

```ts
copyFiles(bundlesDir, `${packageName}?(-*).umd.js?(.map)`, join(releasePath, 'bundles'));
```

This way we can ensure that the bundles only for the current package are copied. Just in case there are other bundles inside of the shared `bundles/` folder.